src/interfaces/IBPool.sol

```diff
@@ -9,6 +9,14 @@ interface IBPool is IERC20 {
     uint256 index; // internal
     uint256 denorm; // denormalized weight
   }
+  /**
+   * @notice Emitted when a swap is performed
+   * @param caller The caller of the swap function
+   * @param tokenIn The address of the token being swapped in
+   * @param tokenOut The address of the token being swapped out
+   * @param tokenAmountIn The amount of tokenIn being swapped in
+   * @param tokenAmountOut The amount of tokenOut being swapped out
+   */
 
```

Comment:
why this space?

Reply:
👀

Reply:
why is the comment touching the struct above, but has a space with the event below (to which it describes)?

Reply:
sorry I fixed it 2 times already, I think the linter is messing with it

Reply:
do you see it correctly now?

Reply:
solhint-plugin-prettier is somewhat opinionated about the spacing between comments, function definitions and function declarations. we could give it a go to solve most of these things programatically (might have to disable some rule on foundry-fmt tho)